plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/WorkspaceFolderNodePathProvider.java

```diff
@@ -0,0 +1,22 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2015 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.plugin.docker.machine;
+
+import com.google.inject.assistedinject.Assisted;
+
+/**
+ * Finds path to workspace folder on docker node.
+ *
+ * @author Alexander Garagatyi
+ */
+public interface WorkspaceFolderNodePathProvider {
```

Comment:
I am not sure about Node as a part of the interface name. 
Is it `WorkspaceFolderNode` or `NodePath`. I think it would be better to move it to `.node` package.

@garagatyi, @skabashnyuk what do you think?